Helpers/RLMRealm+Notifications.m

```diff
@@ -65,4 +65,26 @@ - (void)deleteObjectsWithNotification:(id<NSFastEnumeration>)array
     [self deleteObjects:array];
 }
 
+#pragma mark - private helpers
+
+- (BOOL)rbq_containsObject:(RLMObject *)object {
+    if (!object) {
+        return NO;
+    }
+    
+    Class objectClass = [object class];
```

Comment:
good point.  I wonder if it makes more sense to move this method to RLMRealm+Utilities to avoid duplication and make it more like:

```
- (BOOL)isContainedInRealm:(RLMObject *)realm {
    if (!realm || !object.objectSchema.primaryKeyProperty) {
		return NO;
	}
	
    id primaryKeyValue = nil;
	primaryKeyValue = [RLMObject primaryKeyValueForObject:self];
    
    if (primaryKeyValue) {
    	NSString *objectClassName = [RLMObject classNameForObject:object];
		Class objectClass = NSClassFromString(objectClassName);
        return !![objectClass objectInRealm:realm forPrimaryKey:primaryKeyValue];
    } else {
		return NO;
	}
}
```

I was hoping to avoid including the utilities category but with that much code duplication this seems like the lesser of 2 evils.